### PR TITLE
docs(v4): update changesets

### PR DESCRIPTION
**Pull request by salihozdemir**
Update changesets to make more clear.

### CODE REVIEW

File: .changeset/early-bananas-reflect.md [hunk 1]
```diff
@@ -2,4 +2,8 @@
 "@pankod/refine-core": minor
 ---
 
-Add support [audit log](https://refine.dev/docs/api-reference/core/providers/audit-log-provider/) support for bulk actions hooks (useCreateMany, useDeleteMany, useUpdateMany). 
\ No newline at end of file
+Added [audit log](https://refine.dev/docs/api-reference/core/providers/audit-log-provider/) support for the following hooks:
+
+-   `useCreateMany`
+-   `useDeleteMany`
+-   `useUpdateMany`
```
Comment (omeraplak) [inline .changeset/early-bananas-reflect.md:9]:
```suggestion
-   [`useCreateMany`](https://refine.dev/docs/api-reference/core/hooks/data/useCreateMany/)
-   [`useDeleteMany`](https://refine.dev/docs/api-reference/core/hooks/data/useDeleteMany/)
-   [`useUpdateMany`](https://refine.dev/docs/api-reference/core/hooks/data/useUpdateMany/)
```

File: .changeset/great-moose-jump.md [hunk 2]
```diff
@@ -2,6 +2,4 @@
 "@pankod/refine-antd-audit-log": minor
 ---
 
-`options` property of `<Refine>`'s `resources` property is deprecated. Added `meta` property instead.
-
-`useLogList` updated to use `meta` property instead of `options` property.
+Used new props with deprecated props to ensure backward compatibility.
```
Comment (BatuhanW) [inline .changeset/great-moose-jump.md:5]: Let's be more explicit here. Which deprecated props changed to which props.

File: .changeset/hip-waves-share.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 "@pankod/refine-antd": minor
 ---
 
--   Deprecated `sort` props of `useCheckboxGroup` hook. Use `sorters` prop instead.
+-   `sort` props of `useCheckboxGroup` hook is deprecated. Use `sorters` prop instead.
```
Comment (BatuhanW) [inline .changeset/hip-waves-share.md:5]:
```suggestion
-   `useCheckboxGroup`'s `sort` prop is deprecated. Use `sorters` prop instead.
```

File: .changeset/hip-waves-share.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ useCheckboxGroup({
 })
 ```
 
--   Deprecated `sort` props of `useSelect` hook. Use `sorters` prop instead.
+-   `sort` props of `useSelect` hook is deprecated. Use `sorters` prop instead.
```
Comment (BatuhanW) [inline .changeset/hip-waves-share.md:14]:
```suggestion
-   `useSelect`'s `sort` prop is deprecated. Use `sorters` prop instead.
```

File: .changeset/hip-waves-share.md [hunk 5]
```diff
@@ -20,7 +20,7 @@ useSelect({
 })
 ```
 
--   Deprecated `sort` props of `useRadioGroup` hook. Use `sorters` prop instead.
+-   `sort` props of `useRadioGroup` hook is deprecated. Use `sorters` prop instead.
```
Comment (BatuhanW) [inline .changeset/hip-waves-share.md:23]:
```suggestion
-   `useRadioGroup`'s `sort` prop is deprecated. Use `sorters` prop instead.
```

File: .changeset/hip-waves-share.md [hunk 6]
```diff
@@ -29,7 +29,7 @@ useRadioGroup({
 })
 ```
 
--   Deprecated `resourceName` props of `useImport`hook. Use `resource` prop instead.
+-   `resourceName` props of `useImport` hook is deprecated. Use `resource` prop instead.
```
Comment (BatuhanW) [inline .changeset/hip-waves-share.md:32]:
```suggestion
-   `useImport`'s `resourceName` prop is deprecated. Use `resource` prop instead.
```

File: .changeset/lazy-crews-poke.md [hunk 7]
```diff
@@ -2,12 +2,75 @@
 "@pankod/refine-react-table": minor
 ---
 
--   `initialCurrent` and `initialPagesize` properties of `useTable` hook are deprecated. Use `pagination.current` and `pagination.pageSize` instead. However, for backward compatibility, `initialCurrent` and `initialPagesize` properties will be used if `pagination.current` and `pagination.pageSize` are not provided.
+`useTable` return values and properties are updated.
 
--   `sorter` and `setSorter` return values of `useTable` hook are deprecated. Use `sorters` and `setSorters` instead. However, for backward compatibility, `sorter` and `setSorter` are safe to use.
+-   `initialCurrent` and `initialPageSize` props are deprecated. Use `pagination` prop instead. To ensure backward compatibility, `initialCurrent` and `initialPageSize` props work as before.
```
Comment (BatuhanW) [inline .changeset/lazy-crews-poke.md:7]:
```suggestion
-   `initialCurrent` and `initialPageSize` props are now deprecated. Use `pagination` prop instead.
-   To ensure backward compatibility, `initialCurrent` and `initialPageSize` props will work as before.
```

File: .changeset/lazy-crews-poke.md [hunk 8]
```diff
@@ -2,12 +2,75 @@
...
+    +        current,
+    +        pageSize,
+    +    },
+    })
+    ```
 
-    -   When `pagination.mode` is "off", all records will be fetched from the API and pagination will be hidden.
-    -   When `pagination.mode` is "client", all records will be fetched from the API and pagination will be handled by client.
-    -   When `pagination.mode` is "server", pagination will be handled by the API using `current` and `pageSize` properties of `pagination` object.
+-   `hasPagination` prop is deprecated. Use `pagination.mode` instead. To ensure backward compatibility, `hasPagination` prop works as before.
```
Comment (BatuhanW) [inline .changeset/lazy-crews-poke.md:20]:
```suggestion
-   `hasPagination` prop is now deprecated. Use `pagination.mode` instead.
-   To ensure backward compatibility, `hasPagination` prop will works as before.
```

File: .changeset/lazy-crews-poke.md [hunk 9]
```diff
@@ -2,12 +2,75 @@
...
+        refineCoreProps: {
+    -      hasPagination,
+    +       pagination: {
+    +           mode: "off" | "server" | "client",
+    +       },
+        },
+    })
+    ```
+
+-   `initialSorter` and `permanentSorter` props are deprecated. Use `sorters.initial` and `sorters.permanent` instead. To ensure backward compatibility, `initialSorter` and `permanentSorter` props work as before.
```
Comment (BatuhanW) [inline .changeset/lazy-crews-poke.md:33]:
```suggestion
-   `initialSorter` and `permanentSorter` props are now deprecated. Use `sorters.initial` and `sorters.permanent` instead.
-    To ensure backward compatibility, `initialSorter` and `permanentSorter` props will work as before.
```

File: .changeset/mean-kangaroos-press.md [hunk 10]
```diff
@@ -2,6 +2,6 @@
 "@pankod/refine-ui-tests": minor
 ---
 
-`options` property of `<Refine>`'s `resources` property is deprecated.
+New props are used instead of deprecated ones in the tests.
```
Comment (BatuhanW) [inline .changeset/mean-kangaroos-press.md:5]: Let's mention which props are being used explicitly.

File: .changeset/modern-socks-kiss.md [hunk 11]
```diff
@@ -2,7 +2,7 @@
 "@pankod/refine-mantine": minor
 ---
 
--   Deprecated `sort` props of `useSelect` hook. Use `sorters` prop instead.
+-   `sort` props of `useSelect` hook is deprecated. Use `sorters` prop instead.
```
Comment (BatuhanW) [inline .changeset/modern-socks-kiss.md:5]:
```suggestion
-   `useSelect`'s `sort` prop is now deprecated. Use `sorters` prop instead.
```

File: .changeset/olive-ants-remain.md [hunk 12]
```diff
@@ -1,8 +1,11 @@
...
+"@pankod/refine-nextjs-router": minor
+"@pankod/refine-remix-router": minor
+"@pankod/refine-cloud": minor
+"@pankod/refine-inferencer": minor
+"@pankod/refine-kbar": minor
+"@pankod/refine-ui-tests": minor
 ---
 
--   `AuthProvider` deprecated and renamed to `LegacyAuthProvider` with refine@4. Component and functions are updated to support `LegacyAuthProvider`.
+`AuthProvider` deprecated and renamed to `LegacyAuthProvider` with refine@4. Component and functions are updated to support `LegacyAuthProvider`.
```
Comment (BatuhanW) [inline .changeset/olive-ants-remain.md:11]:
```suggestion
`AuthProvider` is renamed to `LegacyAuthProvider` with refine@4. Components and functions are updated to support `LegacyAuthProvider`.
```

File: .changeset/orange-tools-obey.md [hunk 13]
```diff
@@ -11,8 +11,35 @@
 "@pankod/refine-strapi-graphql": minor
 "@pankod/refine-strapi-v4": minor
 "@pankod/refine-supabase": minor
+"@pankod/refine-medusa": minor
 ---
 
--   Now, `pagination` property of `getList` method has new `mode` property. It can be "off", "client", and "server". If it is "off" or "client", data provider will not send any pagination related query params to the API. If it is "server", data provider will send pagination related query params to the API.
+-   `metaData` props is deprecated for all data provider methods. Use `meta` prop instead.
```
Comment (BatuhanW) [inline .changeset/orange-tools-obey.md:17]:
```suggestion
-   `metaData` prop is now deprecated for all data provider methods. Use `meta` prop instead.
```

File: .changeset/nasty-worms-drive.md [hunk 14]
```diff
@@ -2,17 +2,45 @@
...
-After:
+And it's done. Now your project uses `refine@4.x.x`.
+
+## 📝 Changelog
+
+All **Chakra UI** components re-exported from `@pankod/refine-chakra-ui` have been removed. You should import them from `@chakra-ui/react` package directly.
+
+If the packages are not installed, you can install them with your package manager:
+
+> You don't have to install all of packages. You can install only the packages that you use.
```
Comment (BatuhanW) [inline .changeset/nasty-worms-drive.md:23]:
```suggestion
> You don't have to install all of these packages below. Only install the packages you use.
```

File: .changeset/orange-tools-obey.md [hunk 15]
```diff
@@ -11,8 +11,35 @@
...
 
--   `hasPagination` property of `getList` method is deprecated. Use `pagination.mode` instead. However, for backward compatibility, `hasPagination` property will be used if `pagination.mode` is not provided.
+    If you provide `metaData` with **refine** hooks or components, the `meta` value will be same as your provided `metaData` value.
```
Comment (BatuhanW) [inline .changeset/orange-tools-obey.md:19]:
```suggestion
    For backward compatibility, we still support `metaData` prop with refine v4.
```

I'm not sure about this one, just a suggestion.

File: .changeset/orange-tools-obey.md [hunk 16]
```diff
@@ -11,8 +11,35 @@
...
+    -    metaData
+    +    meta
+    }) => {
+        ...
+    },
+    ```
+
+-   `sort`, `hasPagination`, and `metaData` paramaters of `getList` method is deprecated. Use `sorters`, `pagination`, and `meta` parameters instead.
+
+    If you provide `sort`, `hasPagination` or `metaData` with **refine** hooks or components, the `sorters`, `pagination.mode` or `meta` value will be same as your provided prop value.
```
Comment (BatuhanW) [inline .changeset/orange-tools-obey.md:32]:
```suggestion
    For backward compatibility, we still support `sort`, `hasPagination` and `metaData` props with refine v4.
```

I'm not sure about this one, just a suggestion.

File: .changeset/poor-kids-live.md [hunk 17]
```diff
@@ -2,37 +2,36 @@
...
-import { Box, NumberField, Stack, Typography } from "@pankod/refine-mui";
-```
-
-After:
-
-```tsx
-import { NumberField } from "@pankod/refine-mui";
-import { Box, Stack, Typography } from "@mui/material";
-```
+> You don't have to install all of packages. You can install only the packages that you use.
```
Comment (BatuhanW) [inline .changeset/poor-kids-live.md:9]:
```suggestion
> You don't have to install all of these packages below. Only install the packages you use.
```

File: .changeset/slow-wasps-remain.md [hunk 18]
```diff
@@ -2,7 +2,7 @@
 "@pankod/refine-core": minor
 ---
 
--   Deprecated `congig` props of `useList` and `useInfiniteList` hooks. Use `sorters`, `filters`, `pagination`, and `hasPagination` props instead.
+-   `congig` prop of `useList` and `useInfiniteList` hooks are deprecated. Use `sorters`, `filters`, `pagination` and `hasPagination` props instead.
```
Comment (BatuhanW) [inline .changeset/slow-wasps-remain.md:5]:
```suggestion
-   `useList` and `useInfiniteList`'s `congig` prop is now deprecated. Use `sorters`, `filters`, `pagination` and `hasPagination` props instead.
```

**Review by BatuhanW — CHANGES_REQUESTED** [review]

**Review by alicanerdurmaz — DISMISSED** [review]

**Review by yildirayunlu — APPROVED** [review]

File: .changeset/strange-cats-sort.md [hunk 19]
```diff
@@ -2,17 +2,47 @@
...
-After:
+And it's done. Now your project uses `refine@4.x.x`.
+
+## 📝 Changelog
+
+All **Mantine** components re-exported from `@pankod/refine-mantine` have been removed. You should import them from Mantine packages directly.
+
+If the packages are not installed, you can install them with your package manager:
+
+> You don't have to install all of packages. You can install only the packages that you use.
```
Comment (BatuhanW) [inline .changeset/strange-cats-sort.md:23]:
```suggestion
> You don't have to install all of these packages below. Only install the packages you use.
```

File: .changeset/strange-knives-end.md [hunk 20]
```diff
@@ -2,7 +2,7 @@
 "@pankod/refine-mui": minor
 ---
 
--   Deprecated `sort` props of `useAutocomplete` hook. Use `sorters` prop instead.
+-   `sort` prop of `useAutocomplete` hook is deprecated. Use `sorters` prop instead.
```
Comment (BatuhanW) [inline .changeset/strange-knives-end.md:5]:
```suggestion
-   `useAutocomplete`'s sort prop is now deprecated. Use `sorters` prop instead.
```

File: .changeset/tidy-actors-grow.md [hunk 21]
```diff
@@ -2,12 +2,67 @@
...
+    +        current,
+    +        pageSize,
+    +    },
+    })
+    ```
 
-    -   When `pagination.mode` is "off", all records will be fetched from the API and pagination will be hidden.
-    -   When `pagination.mode` is "client", all records will be fetched from the API and pagination will be handled by client.
-    -   When `pagination.mode` is "server", pagination will be handled by the API using `current` and `pageSize` properties of `pagination` object.
+-   `hasPagination` prop is deprecated. Use `pagination.mode` instead. To ensure backward compatibility, `hasPagination` prop works as before.
```
Comment (BatuhanW) [inline .changeset/tidy-actors-grow.md:20]:
```suggestion
-   `hasPagination` prop is now deprecated. Use `pagination.mode` instead.
-   To ensure backward compatibility, `hasPagination` prop will work as before.
```

**Review by BatuhanW — CHANGES_REQUESTED** [review]

File: .changeset/great-cooks-sell.md [hunk 22]
```diff
@@ -2,17 +2,83 @@
...
+    -    initialPageSize,
+    +    pagination: {
+    +        current,
+    +        pageSize,
+    +    },
+    })
+    ```
+
+-   `hasPagination` prop is now deprecated. Use `pagination.mode` instead.
+-   To ensure backward compatibility, `hasPagination` prop works as before.
```
Comment (BatuhanW) [inline .changeset/great-cooks-sell.md:33]: Please, let's review **all** sentences like this and rephrase them. 

Example: `-   To ensure backward compatibility, `initialCurrent` and `initialPageSize` props will work as before.`

File: .changeset/lazy-crews-poke.md [hunk 23]
```diff
@@ -2,12 +2,80 @@
...
+    +        pageSize,
+    +    },
+    })
+    ```
 
-    -   When `pagination.mode` is "off", all records will be fetched from the API and pagination will be hidden.
-    -   When `pagination.mode` is "client", all records will be fetched from the API and pagination will be handled by client.
-    -   When `pagination.mode` is "server", pagination will be handled by the API using `current` and `pageSize` properties of `pagination` object.
+-   `hasPagination` prop is now deprecated. Use `pagination.mode` instead.
+-   To ensure backward compatibility, `hasPagination` prop will works as before.
```
Comment (BatuhanW) [inline .changeset/lazy-crews-poke.md:22]: Please, let's review all sentences like this and rephrase them.

Example: -   To ensure backward compatibility, initialCurrentandinitialPageSize props will work as before.

File: .changeset/tidy-actors-grow.md [hunk 24]
```diff
@@ -2,12 +2,72 @@
...
+    +        pageSize,
+    +    },
+    })
+    ```
 
-    -   When `pagination.mode` is "off", all records will be fetched from the API and pagination will be hidden.
-    -   When `pagination.mode` is "client", all records will be fetched from the API and pagination will be handled by client.
-    -   When `pagination.mode` is "server", pagination will be handled by the API using `current` and `pageSize` properties of `pagination` object.
+-   `hasPagination` prop is now deprecated. Use `pagination.mode` instead.
+-   To ensure backward compatibility, `hasPagination` prop works as before.
```
Comment (BatuhanW) [inline .changeset/tidy-actors-grow.md:22]: Please, let's review all sentences like this and rephrase them.

Example: -   To ensure backward compatibility, initialCurrentandinitialPageSize props will work as before.

File: .changeset/young-pears-dream.md [hunk 25]
```diff
@@ -2,19 +2,86 @@
...
+    -    initialPageSize,
+    +    pagination: {
+    +        current,
+    +        pageSize,
+    +    },
+    })
+    ```
+
+-   `hasPagination` prop is now deprecated. Use `pagination.mode` instead.
+-   To ensure backward compatibility, `hasPagination` prop works as before.
```
Comment (BatuhanW) [inline .changeset/young-pears-dream.md:24]: Please, let's review all sentences like this and rephrase them.

Example: -   To ensure backward compatibility, initialCurrentandinitialPageSize props will work as before.

**Review by BatuhanW — CHANGES_REQUESTED** [review]

**Review by BatuhanW — APPROVED** [review]

**Review by yildirayunlu — APPROVED** [review]

**Pull request merged**